modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/runtime/source/ExtensionMessageSource.java

```diff
@@ -115,6 +118,9 @@
 public class ExtensionMessageSource extends ExtensionComponent<SourceModel> implements MessageSource,
     ExceptionCallback<ConnectionException>, ParameterizedSource, ConfiguredComponent, LifecycleStateEnabled {
 
+  private static final Boolean SHOULD_HANDLE_ACCESS_TOKEN_EXPIRED_EXCEPTIONS_ON_SOURCES =
```

Comment:
Maybe it would be better to have its name saying IGNORE_ACCESS_TOKEN_EXPIRED_EXCEPTION_NOTIFICATIONS_ON_SOURCES , and if the property is present, ignore the handling.